### ISSUE TITLE
The args ` clip` & `preserve_paper`  are missing in the pip library

### ISSUE THREAD

**Issue by ustcjerry** [jrosebr1/color_transfer#8]
When I use following line to install the lib,
`pip install color_transfer`
and use codes in the `example.py`,  there will be errors:
`TypeError: color_transfer() got an unexpected keyword argument 'preserve_paper'`
``TypeError: color_transfer() got an unexpected keyword argument 'clip'``

so does the files in the pip lib older than the current repo?

**Comment by jrosebr1** [comment]
Yes, the version on PyPI is much older. Clone this repo down and install via `setup.py`

**Comment by ustcjerry** [comment]
Please considering to update the PyPI version ヽ(￣▽￣)ﾉ
